tests/alembic_migrations/versions/v_7b139906ac46_add_not_null_constraint_on_meta_vars/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from sqlalchemy.orm import sessionmaker
+from ....helpers import insert_crmp_data
+
+
+@pytest.fixture(scope="module")
+def target_revision():
+    return "7b139906ac46"
+
+
+@pytest.fixture(scope="function")
+def sesh_with_large_data(prepared_schema_from_migrations_left):
+    engine, script = prepared_schema_from_migrations_left
+    sesh = sessionmaker(bind=engine)()
+    insert_crmp_data(sesh)
```

Comment:
If you omit this you will have a clean empty database. That's probably desirable for this test.

Reply:
so remove this entire function from ```conftest.py```?